test/extended/util/annotate/rules.go

```diff
@@ -304,6 +304,11 @@ var (
 			`Unidling should work with TCP`,
 			`Unidling should handle many TCP connections`,
 		},
+		// tests that don't pass under Cilium
+		"[Skipped:Network/Cilium]": {
+			// Cilium does not support services with internal traffic local policy
+			`internalTrafficPolicy=Local`,
```

Comment:
This should have a link to the relevant Cilium issue so people in the future can know whether this is still needed.

Reply:
done